tsdb/blockwriter.go

```diff
@@ -39,6 +39,9 @@ type BlockWriter struct {
 	chunkDir  string
 }
 
+// ErrNoSeriesAppended throws an error if the series count is zero while flushing blocks.
```

Comment:
I would avoid the word "throw". Go doesn't have exceptions to throw, it just returns errors ("Errors are values" is one of the Go proverbs, IIRC.)

 So how about: "ErrNoSeriesAppended is returned if the series count is zero while flushing blocks."

Reply:
Sg 👍